src/components/hint/_hint.scss

```diff
@@ -16,14 +16,35 @@
   // Reduces margin-bottom of hint when used after the default label (no class)
   // or govuk-label--s for better vertical alignment.
 
-  // This adjustment will not work when the label is inside the <h1>, however
-  // it is unlikely that the default or govuk-label--s class would be used in
-  // this case.
+  // This adjustment will not work when the label is inside the <h1>, however it
+  // is unlikely that the default or govuk-label--s class would be used in this
+  // case.
 
-  // This adjustment will not work in browsers that do not support :not()
-  // Users with these browsers will see the default size margin (5px larger)
+  // This adjustment will not work in browsers that do not support :not() Users
```

Comment:
Can we add a full stop here to keep these sentences from running into each other like this?

Reply:
adjusted